### PR TITLE
Exemptions Framework

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17418

*Description of changes:*
- Implements routes and placeholder templates
- Add home view with button footer, fee summary and stepper
- Basic state model, actions and getters
- Updated breadcrumb/fee summary

- Updated LD Default Logic

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1552-i3fyse9f.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1552-i3fyse9f.web.app

**Review by doug-lovett — APPROVED** [review]
Ha ha, small PR. Looks great.

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -386,6 +386,24 @@
               </v-btn>
             </template>
             <v-list class="actions__more-actions registration-actions">
+              <v-list-item
+                v-if="areExemptionsEnabled"
```
Comment (dimak1) [inline ppr-ui/src/components/tables/common/TableRow.vue]: Small but I would prefer `isExemptionsEnabled` for consistency.

**Review by dimak1 — APPROVED** [review]
Looks great!

File: ppr-ui/src/components/tables/common/TableRow.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/tables/common/TableRow.vue]: For sure, can update to `isEmemptionEnabled` 👍

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1552-i3fyse9f.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1552-i3fyse9f.web.app

**Pull request merged**